### PR TITLE
Maintenance: Security updates

### PR DESCRIPTION
**Pull request by MKodde**
1. Yarn & Composer updates installed
2. Upgraded Webpack Encore from v1 to 4

### CODE REVIEW

**Comment by MKodde** [comment]
The security updates are now breaking the tests

I've investigated on my dev env and the following things are in the way:

1. Chrome and or Chromedriver is not installed correctly. Ive fixed this on my dev machine. But it takes some more effort to land those changes in this project. And should not be part of this PR
2. Behat tests fail on Symfony errors that the ErrorController is fishing services directly from the container using `$this->get`. That should be fixed in the stepup-bundle. Also not for this PR.

I feel OK to move forward with this, and tag it pre-release for now. We are about to overhaul this project and upgrade it to SF 5/6. So putting in this work now seems illogical.

File: webpack.config.js [hunk 1]
```diff
@@ -5,7 +5,11 @@ Encore
     .setPublicPath('/build')
     .cleanupOutputBeforeBuild()
     .addStyleEntry('global', './public/scss/application.scss')
-    .addLoader({ test: /\.scss$/, loader: 'import-glob-loader' })
+    .addLoader({test: /\.scss$/, loader: 'webpack-import-glob-loader'})
+    .configureLoaderRule('eslint', loaderRule => {
+        loaderRule.test = /\.(jsx?|vue)$/
```
Comment (phavekes) [inline webpack.config.js:10]: What does this do?

File: package.json [hunk 2]
```diff
@@ -1,17 +1,20 @@
 {
     "browserslist": "last 2 versions, ie 11, > 1%",
     "devDependencies": {
-        "@symfony/webpack-encore": "^1",
-        "compass-mixins": "^0.12.10",
+        "@babel/core": "^7.20.5",
+        "@babel/preset-env": "^7.20.2",
+        "@symfony/webpack-encore": "^4",        "compass-mixins": "^0.12.10",
```
Comment (phavekes) [inline package.json:6]: Indent error?

File: webpack.config.js (hunk 1 above)
Comment (MKodde) [reply webpack.config.js]: Thanks for pointing this out. 

To answer your question. This loads eslint and configures it to mark jsx and vue files a test resources

And that is completely out of scope for this project. Removed this loader.

FYI This code came from another stepup project where we already upgraded to Encore 4. So I yanked some of the updated config from there. But kept a bit too much :blush:

File: package.json (hunk 2 above)
Comment (MKodde) [reply package.json]: Yes, fixed that :+1:

**Pull request merged**